nodes/geometry/point_inside_camera_frustrum.py

```diff
@@ -0,0 +1,40 @@
+import bpy
+from bpy.props import *
+from ... base_types.node import AnimationNode
+import bpy_extras
+
+class PointInCameraFrustrumNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_PointInCameraFrustrumNode"
+    bl_label = "Point in Camera Frustrum"
+    
+    def create(self):
+        self.newInput("Scene", "Scene", "scene", hide = True)
+        self.newInput("Object", "Camera", "camera")
+        self.newInput("Vector", "Point", "point", value=(0, 0, 0))
+        self.newInput("Float", "Threshold", "threshold", value = 0.0)
+        
+        self.newOutput("Float", "Image u", "u")
+        self.newOutput("Float", "Image v", "v")
+        self.newOutput("Float", "Z depth", "z")
+        self.newOutput("Boolean", "Visible", "visible")
+        
+    def getExecutionCode(self):
```

Comment:
is there any particular reason you use the ``getExecutionCode`` method here instead of just ``execute``?